clients/rook/Dockerfile.release-prebuilt

```
@@ -0,0 +1,35 @@
+# syntax=docker/dockerfile:1
+
+FROM busybox:latest AS prep
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)